Formula/libsmi.rb

```diff
@@ -6,9 +6,13 @@ class Libsmi < Formula
   sha256 "f21accdadb1bb328ea3f8a13fc34d715baac6e2db66065898346322c725754d3"
   license all_of: ["TCL", "BSD-3-Clause", "Beerware"]
 
+  # The directory listing page where the tarballs were found now gives a 403
```

Comment:
How about asking upstream to fix that?

Reply:
I was hoping to avoid having to post on their mailing list (you have to subscribe to do so) but I guess we _should_ ask 😆

Let me create an email alias and I'll see if we get any response (last post on the mailing list was 2022-01 but it did get a response).

Reply:
another idea, maybe we can also use gentoo to check the version. 

For the forbidden response, let me reach out to them :)